packages/manager/modules/pci/src/projects/project/data-processing/notebooks/data-processing-notebooks.html

```diff
@@ -0,0 +1,144 @@
+<div data-ui-view>
+    <div>
+        <!-- cui messages -->
+        <cui-message-container
+            data-messages="$ctrl.messages"
+        ></cui-message-container>
+        <oui-datagrid
+            data-rows="$ctrl.notebooks"
+            data-refreshable
+            data-criteria="$ctrl.criteria"
+            data-on-criteria-change="$ctrl.onListParamChange($criteria)"
+        >
+            <oui-datagrid-column
+                data-title="::'data_processing_notebook_name' | translate"
+                data-property="spec.name"
+                data-sortable
+                data-type="string"
+                data-searchable
+                data-filterable
+            >
+                <oui-button
+                    data-variant="link"
+                    data-on-click="$ctrl.showNotebook($row.id)"
+                >
+                    <span data-ng-bind="::$row.spec.name"></span>
+                </oui-button>
+            </oui-datagrid-column>
+            <oui-datagrid-column
+                data-title="::'data_processing_region' | translate"
+                data-property="spec.region"
+                data-type="string"
+                data-sortable
+                data-searchable
+                data-filterable
+            >
+                <span
+                    data-ng-bind="::$ctrl.ovhManagerRegionService.getTranslatedMicroRegion($row.spec.region)"
+                ></span>
+            </oui-datagrid-column>
+            <oui-datagrid-column
+                data-title="::'data_processing_framework' | translate"
+                data-property="spec.env.engineName"
+                data-type="string"
+                data-sortable
+                data-searchable
+                data-filterable
+            >
+                <span
+                    data-ng-bind="::$row.spec.env.engineName + ' ' + $row.spec.env.engineVersion"
+                ></span>
+            </oui-datagrid-column>
+            <oui-datagrid-column
+                data-title="::'data_processing_privacy' | translate"
+                data-property=""
```

Comment:
Is it needed ?

Reply:
I kept it as it will be used in the very near future but could be removed